frame.go

```diff
@@ -161,6 +161,7 @@ const (
 
 	// prepare flags
 	flagWithPreparedKeyspace uint32 = 0x01
+	flagLWT                  int    = 0x80000000
```

Comment:
@ManManson can you confirm it's correct. Does 4.0 include it?

Reply:
@mmatczuk  Yes, it is consistent with the value I am planning to add on the scylla side.
AFAIK it's not to be included to 4.0, but should be backported to some later version.

Right at the moment the corresponding issue (https://github.com/scylladb/scylla/issues/6259) is tagged with `4.3` label.